modules/tinymce/src/core/main/ts/api/dom/EventUtils.ts

```diff
@@ -205,52 +208,21 @@ class EventUtils {
         continue;
       }
 
-      // Handle mouseenter/mouseleaver
-      if (!self.hasMouseEnterLeave) {
-        fakeName = self.mouseEnterLeave[name];
-
-        if (fakeName) {
-          nativeHandler = (evt) => {
-            const current = evt.currentTarget;
-            let related = evt.relatedTarget;
-
-            // Check if related is inside the current target if it's not then the event should
-            // be ignored since it's a mouseover/mouseout inside the element
-            if (related && current.contains) {
-              // Use contains for performance
-              related = current.contains(related);
-            } else {
-              while (related && related !== current) {
-                related = related.parentNode;
-              }
-            }
-
-            // dispatch fake event
-            if (!related) {
-              evt = fix(evt || win.event);
-              evt.type = evt.type === 'mouseout' ? 'mouseleave' : 'mouseenter';
-              evt.target = current;
-              self.executeHandlers(evt, id);
-            }
-          };
-        }
-      }
```

Comment:
Removed this as it was broken and all browsers support mouse enter/leave events now.